### PR TITLE
Provider request ui updates

### PR DESCRIPTION
**Pull request by cheshireoctopus**
https://trello.com/c/el7YoizO/75-update-provider-request-page

## Overview

Updates styling of `<ProviderPage />` to align with [spec](https://www.figma.com/file/hI9oTtobFk8PQ9pzJuZbqG/Feed-a-Doc?node-id=21816%3A29657).

image

### CODE REVIEW

File: app/javascript/packs/pages/ProviderPage.jsx [hunk 1]
```diff
@@ -87,57 +122,70 @@ export default function ProviderPage(props) {
...
-        ) : (
-          <Typography component="h1" align="center" className={classes.complete} gutterBottom>
-            The provider has marked this request as complete.
-          </Typography>
-        )}
-      </Container>
-    </Paper>
+      <Box className={classes.requestHeader}>
+        <Typography component="p" variant="h6" align="center">
+          Find more care providers to help <Link to="/browse" className={classes.browseLink}>here.</Link>
```
Comment (cantino) [inline app/javascript/packs/pages/ProviderPage.jsx:183]: Let's hide this link for the MVP until /browse is ready.

File: app/javascript/packs/pages/ProviderPage.jsx [hunk 2]
```diff
@@ -87,57 +122,70 @@ export default function ProviderPage(props) {
...
+        </Typography>
+      </Box>
+      <Box className={classes.footer}>
+        <Typography component="p" align="center" gutterBottom>
+          <b>Need help editing or removing a request?</b>
+        </Typography>
+        <Typography component="p" align="center" gutterBottom>
+          Please email <a href="mailto:covidcarenetwork@gmail.com">covidcarenetwork@gmail.com</a> to report requests that should be updated or removed.
```
Comment (cantino) [inline app/javascript/packs/pages/ProviderPage.jsx]: Please change to support@feedadoc.zendesk.com (right @mavenkeith ?)

**Review by cantino — CHANGES_REQUESTED** [review]

File: app/javascript/packs/pages/ProviderPage.jsx (hunk 1 above)
Comment (cheshireoctopus) [reply app/javascript/packs/pages/ProviderPage.jsx:183]: Done 👍

File: app/javascript/packs/pages/ProviderPage.jsx (hunk 2 above)
Comment (cheshireoctopus) [reply app/javascript/packs/pages/ProviderPage.jsx]: Changed 👍

**Review by cantino — APPROVED** [review]

**Pull request merged**